src/ol/source/Tile.js

```diff
@@ -34,7 +34,7 @@ import {scale as scaleSize, toSize} from '../size.js';
  * @property {import("../proj.js").ProjectionLike} [projection] Projection.
  * @property {import("./State.js").default} [state] State.
  * @property {import("../tilegrid/TileGrid.js").default} [tileGrid] TileGrid.
- * @property {boolean} [wrapX=true] WrapX.
+ * @property {boolean} [wrapX=false] WrapX.
```

Comment:
This is also a documentation bug.  The tile source doesn't apply a default and the parent defaults to false 
https://github.com/openlayers/openlayers/blob/79b522990811e4ada0bb845e8c0920e20a772a89/src/ol/source/Source.js#L88-L92